spec.bs

```diff
@@ -804,10 +804,11 @@ On the other hand, methods for getting data from the [=shared storage database=]
     1. Let |node| be |sourceDocument|'s [=node navigable=].
     1. While |node| is not null:
         1. If |node| has a "fenced frame config struct":
-            1. Let |origin| be |node|'s [=active document=]'s [=document/origin=].
+            1. Let |site| be the result of running [=obtain a site=] with |node|'s [=active document=]'s [=document/origin=].
+            1. [=Assert=] that |site| is not an [=opaque origin=].
```

Comment:
Why can't |node| have an opaque origin?

Reply:
Good point. I removed that assumption, and put in a condition to skip the case where it's opaque.